### ISSUE TITLE
NVDA fails to check for updates

### ISSUE THREAD

**Issue by kyleman** [nvaccess/nvda#19443]
### Brief summary

I can no longer get updates with any 2026 NVDA build. I am on a corporate laptop with a corporate HTTPS certificate installed.

### Steps to reproduce

* Use the check for update option in the help menu
* A dialogue pops up saying, "Error checking for update. Unable to establish a connection to the NV Access server."

### Actual behavior

NVDA+n
NVDA  menu
h
User Guide  u
c
Checking for Update  dialog  Checking for update
Error  dialog  Error checking for update.
Unable to establish a connection to the NV Access server.
OK  button

### Expected behavior

NVDA is able to properly check for updates.

### NVDA logs, crash dumps and other attachments

Error checking for update
Traceback (most recent call last):
  File "urllib\request.pyc", line 1319, in do_open
  File "http\client.pyc", line 1358, in request
  File "http\client.pyc", line 1404, in _send_request
  File "http\client.pyc", line 1353, in endheaders
  File "http\client.pyc", line 1113, in _send_output
  File "http\client.pyc", line 1057, in send
  File "http\client.pyc", line 1499, in connect
  File "ssl.pyc", line 455, in wrap_socket
  File "ssl.pyc", line 1076, in _create
  File "ssl.pyc", line 1372, in do_handshake
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Missing Authority Key Identifier (_ssl.c:1032)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "updateCheck.pyc", line 244, in checkForUpdate
  File "urllib\request.pyc", line 189, in urlopen
  File "urllib\request.pyc", line 489, in open
  File "urllib\request.pyc", line 506, in _open
  File "urllib\request.pyc", line 466, in _call_chain
  File "urllib\request.pyc", line 1367, in https_open
  File "urllib\request.pyc", line 1322, in do_open
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Missing Authority Key Identifier (_ssl.c:1032)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "updateCheck.pyc", line 389, in _bg
  File "updateCheck.pyc", line 252, in checkForUpdate
  File "updateCheck.pyc", line 1102, in _updateWindowsRootCertificates
ctypes.ArgumentError: argument 2: TypeError: expected LP_c_byte instance instead of bytes
 

### NVDA type

portable copy

### NVDA version

beta-54074,0ec178ae (2026.1.0.54074)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Every 2026 version i use exhibits the same behavior.

### Windows version

Version; Value: 10.0.22631 Build 22631

### Name and version of other software in use when reproducing the issue

n/a

### Other information about your system

n/a

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by SaschaCowley** [comment]
@kyleman can you try [this test build](https://download.nvaccess.org/snapshots/try/try-i19443/nvda_snapshot_try-i19443-54119,63299422.exe) and let us know if it fixes your issue?
@neurrone, @hwf1324 tagging you because you've indicated you're experiencing #15905, so you may also be able to test this. If you're able, could you try checking for updates from a recent alpha version, and if you see this issue, try again with the test build linked above?

**Comment by hwf1324** [comment]
I cannot test it because it may require an SSL certificate to handle traffic from nvaccess.org.

**Comment by Neurrone** [comment]
I'm no longer in a corporate network that does https interception.

**Comment by SaschaCowley** [comment]
@hwf1324 what do you mean?

**Comment by hwf1324** [comment]
I encountered #15905 because a program was using self-signed certificates to intercept traffic from github.com.
However, no program is using self-signed certificates to intercept traffic from nvaccess.org.

**Comment by SaschaCowley** [comment]
I see, thanks anyway, @hwf1324

**Comment by kyleman** [comment]
I tested the linked build, but there doesn't seem to be a check for updates in the help menu. @SaschaCowley

**Comment by SaschaCowley** [comment]
Hi @kyleman, whoops, my bad. Could you try [this test build](https://download.nvaccess.org/snapshots/try/try-i19443/nvda_snapshot_try-i19443-54246,566d6fea.exe)?
